### ISSUE TITLE
Change subtitles format. 

### ISSUE THREAD

**Issue by Beech0408** [JunkFood02/Seal#500]
### Checklist

- [ ] This feature I'm requesting is already implemented in yt-dlp.
- [ ] This feature is merely a UI/UX update.
- [ ] This feature is suitable for primary users with little knowledge about yt-dlp.
- [ ] This feature is available for most websites, not only the video platform I use.
- [X] This feature is suitable for a large variety of videos.
- [X] This feature is not going to conflict with many of the existing options.

### Is your feature request related to a problem? Please describe.

Whenever I download videos with seal, using the "Embed subtitles" option, it always produces its subtitles in .vtt format. 

### Describe the solution you'd like

Is there to change the subtitles format to another format like .srt or .ass format before downloading?

### Additional context

The reason for this question is because the device I use to watch the YT videos I download does not show the .vtt subs when played, and I cannot install VLC player in the device.
I await your response.
Thanks in advance.

**Comment by Marco-9456** [comment]
For now you can use the following commands;
--convert-subs `FORMAT`
--sub-langs `en`
--write-sub
--embed-subs
--compat-option no-keep-subs

>**Note**
currently supported: ass, lrc, srt, vtt

**Comment by Beech0408** [comment]
Please do not be angry. I don't know how to do this. Please it would be so nice if you could show me how to do this through an instructional video.
I would be very grateful if you could.
Thanks in advance.
Please I await your response.

**Comment by Marco-9456** [comment]
>how to do this through an instructional video. I would be very grateful if you could. Thanks in advance. Please I await your response.


https://user-images.githubusercontent.com/41828058/211543647-9ef3e066-3bf4-4782-b127-d4db02503e42.mp4

**Comment by venkat2305** [comment]
@Marco-9456 hey, if you don't mind, can you please share your custom command templates?

**Comment by Marco-9456** [comment]
> @Marco-9456 hey, if you don't mind, can you please share your custom command templates?

No

**Comment by Marco-9456** [comment]
> @Marco-9456 Thank you so very much for all your help. I cannot express my gratitude to you enough. I wish you a happy and prosperous new year!

Thanks

**Comment by Marco-9456** [comment]
@Beech0408  Please close this issue as completed if my answer solved the problem.

**Comment by Beech0408** [comment]
Sorry, but I am afraid that it didn't. I did as exactly as you instructed in your screen recorded video when I was downloading a YT video less than 90mb. However, it took a lot of time writing the subtitles and when I canceled the process out of frustration, I opened my File Manager and checked the folder for Seal, only to find out that there was about 2 split .webm files of the video I wanted to download, each ranging from at least 400 MB.
You cannot fathom how sad and frustrated I was.
I'm sorry that your advice didn't work out. If it did, I would have closed this issue.
